framework/tests/value/Issue1579.java

```diff
@@ -0,0 +1,13 @@
+// Test case for Issue 1579
+// https://github.com/typetools/checker-framework/issues/1579
+
+public class Issue1579 {
+    public int[][] method(int[] array1, int[] array2) {
+        // Required for crash
+        for (int i = 0; i < array1.length; i++) {}
+
+        //TODO: This shouldn't issue an error.
```

Comment:
I've got another branch that fixes this warning.  I'll post a pull request after this one is merged.